frontend/src/properties/PropertiesWebSocketContainer.tsx

```diff
@@ -78,11 +78,7 @@ export const PropertiesWebSocketContainer = ({
   const [{ value, context }, dispatch] = useMachine<
     PropertiesWebSocketContainerContext,
     PropertiesWebSocketContainerEvent
-  >(propertiesWebSocketContainerMachine, {
-    context: {
-      currentSelection: selection,
-    },
-  });
+  >(propertiesWebSocketContainerMachine);
```

Comment:
I don't mind having this change temporarily but as soon as you're done with the first version of the validation rules, the lifecycle of the properties will needs to be updated to fix this regression.

Reply:
Issue https://github.com/eclipse-sirius/sirius-components/issues/602 created